cmd/security-secretstore-setup/secretstore-init.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/dumb-init /bin/sh
```

Comment:
@anonymouse64 Can we declare this as docker-specific file or can you suggest a docker-specific version of the file?  I asked @tingyuz for this hook so that we can spawn the token-issuing server as a subprocess and have proper process reaping and signal handling.  cc: @jim-wang-intel 

See also https://github.com/Yelp/dumb-init/blob/master/README.md
See also Vault's boilelplate: https://github.com/hashicorp/docker-vault/tree/master/0.X

Reply:
I guess before we go further with figuring out what to do with `dumb-init` here, my larger concern is about this being a shell script at all... See my comment in the actual review

Reply:
This is no longer valid as we decide to remove script.